src/features/hotspots/setup/hotspotSetupSlides.ts

```diff
@@ -1,26 +1,25 @@
-import Icon from '@assets/images/placeholder.svg'
 import { CarouselItemData } from '../../../components/CarouselItem'
 
 const slides: Array<CarouselItemData> = [
   {
     title: 'hotspot_setup.education.cards.0.title',
     desc: 'hotspot_setup.education.cards.0.subtitle',
-    Icon,
+    image: require('../../../assets/images/educationScreen/outside.jpg'),
```

Comment:
Is it just me or does this image look pixelated? Are you sure it's the highest resolution available in the source?